### DISCUSSION TITLE
Advanced log screen and no rcvd QSL

### DISCUSSION THREAD

**Discussion by soakland5** [magicbug/Cloudlog#1943] [Q&A] [answered]
When I look at the same QSO and QSL data on both the overview and advanced screens, I see that it shows a GREEN arrow as LOTW confirmed in the OVERVIEW screen but the QSL rcvd column says "NO" in the Advanced screen.
 
 It's odd because other callsigns for that particular station location show in both. But the Main callsign (active) doesn't.

Has anyone else but me experienced this? Am I doing something wrong?

Thanks
Scott W1AL

**Comment by AndreasK79** [comment]
You are not doing anything wrong. The advanced screen is only showing QSL status. The original thought behind this, was a QSL management system, and more a proof of concept.

I took the original code and expanded it quite a lot (and turned it into an advanced logbook). I never changed the QSL part of it, but it should really reflect QSL/LoTW/eQSL status.

**Comment by AndreasK79** [comment, marked as answer]
@soakland5 I pushed an update to the advanced logbook showing QSL/LoTW/eQSL like the regular logbook. This is currently only in the dev branch, and will be available in master when a new release is issued.

**Comment by soakland5** [comment]
Terrific ! Thanks for the update

**Comment by soakland5** [comment]
Hi, it doesn’t come up in the web version yet

**Comment by phl0** [comment]
As @AndreasK79 mentioned it is currently in dev branch only. You are on master probabaly. So you will get this once Cloudlog gets a new release (merging features from dev to master branch).
